### DISCUSSION TITLE
How I Use Meta Bind to Create Interactive Buttons in Obsidian

### DISCUSSION THREAD

**Discussion by motion2082** [mProjectsCode/obsidian-meta-bind-plugin#449] [Show and tell]
**How I Use Meta Bind to Create Interactive Buttons in Obsidian** 

In today's video I'm going to showcase how I use Meta Bind Buttons inside Obsidian. I'll share examples of buttons you can create with the Meta Bind Plugin showcasing the Meta Bind Button Builder as well as some custom CSS buttons. 

If you're an Obsidian user who is wanting to make buttons for your Obsidian Vault then this video is definitely for you. Join me as create some useful buttons inside Obsidian. 

Video
https://www.youtube.com/watch?v=dP33gV_R1wo&t=498s

![Thumbnail2_8 1 1](https://github.com/user-attachments/assets/d5ebb944-9da6-4bc9-b8f6-c5a53ef631f0)

**Comment by mProjectsCode** [comment]
Nice video 👍
